aws/table_aws_ecs_container_instance.go

```diff
@@ -0,0 +1,226 @@
+package aws
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsEcsContainerInstance(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ecs_container",
+		Description: "AWS ECS Container",
+		List: &plugin.ListConfig{
+			ParentHydrate: listEcsClusters,
+			Hydrate:       listEcsContainerInstances,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "container_instance_arn",
+				Description: "The namespace Amazon Resource Name (ARN) of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContainerInstanceArn"),
+			},
+			{
+				Name:        "instance_id",
+				Description: "The EC2 instance ID of the container instance.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ec2InstanceId"),
+			},
+			{
+				Name:        "agent_connected",
+				Description: "True if the agent is connected to Amazon ECS.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("AgentConnected"),
```

Comment:
@RyanJarv I believe we can remove this `transform.FromField(...)` line, and others below, where the property name can be snake cased into the intended column name correctly.

We usually need the `transform.FromField` transforms when the snake cased property name doesn't resolve to the column name, either due to different naming, e.g., `ContainerInstanceArn` -> `arn`, or if the property name contains something like `ID` that throws off snake casing.